### DISCUSSION TITLE
[UPDATE] Production APIs are in protection right now!

### DISCUSSION THREAD

**Discussion by NYT92** [amvstrm/amvstrm#43] [Announcements]
**UPDATE: We are now turning off IUAM and we also still blocking everyone from accessing the old API URL. Please use our official API and wait till we turn off the mode.**

------

We are currently experiencing a DDOS attack, and 2 of our production APIs might be inaccessible because of Cloudflare. Please be patient before we give you guys an update and I am sorry for the disruption too because we just saw those suspicious traffic spikes.

![image](https://github.com/amvstrm/amvstrm/assets/53612429/508cef5b-439c-48f3-a03a-098ae0ca0f28)

**Comment by rishabnotfound** [comment]
Bro when will be that old api will get live again? Any guess?

**Reply by NYT92** [reply]
Yes, i will turn off protection tomorrow and also the old domain is a mirror to this API ``api.amvstr.me`` since still many people are still using it that is why we keep it but now we might consider asking people to switch to api.amvstr.me in the future.

**Comment by iotserver24** [comment]
i want to mirror your api to cloudflare workers(new api not old)
pls can you say how

**Reply by NYT92** [reply]
You can do that but I just don't like cloudflare worker limitations since there are tons of requests that the website needs to fetch.

**Reply by ImmortalXnetwork** [reply]
when will be api.amvstr.me is going to alive

**Reply by iotserver24** [reply]
> when will be api.amvstr.me is going to alive

hey upload your own api rn 
i did same for now
